src/main/kotlin/com/terraformation/backend/tracking/db/ObservationResultsStore.kt

```diff
@@ -64,6 +65,21 @@ class ObservationResultsStore(private val dslContext: DSLContext) {
     return fetchByCondition(OBSERVATIONS.PLANTING_SITE_ID.eq(plantingSiteId), limit)
   }
 
+  fun fetchLatestPerSubzone(
```

Comment:
On some more thought, I will build out more complete aggregate models and combine observation results and stats from the bottoms up.